package.json

```diff
@@ -76,12 +77,19 @@
     "@storybook/manager-webpack5": "6.3.12",
     "@storybook/preset-scss": "^1.0.3",
     "@storybook/react": "6.3.12",
+    "@swc/core": "^1.2.133",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
Can any of these new packages be moved to devDependencies?

Reply:
I thinks these dependencies are in devDependencies already